Gemfile

```diff
@@ -46,7 +48,10 @@ end
 
 group :development, :test do
   gem 'brakeman', require: false
-  gem 'bullet'
+  # TODO: Bullet does not yet support Rails 6.1.3.1, but that Rails version is required that ActiveStorage
+  # works properly with the latest Google Cloud Services
+  # => Reenable as soon as support was added
+  # gem 'bullet'
```

Comment:
Problem was:
Either I upgrade Rails and can use the latest active storage version which works correctly with gcs
or
I downgrade to the last working solution and keep Bullet enabled.

Reply:
Obviously went for former option